backend/src/routes/reports/rpt_PA_ChangeRequestTypesFYSummary.js

```diff
@@ -0,0 +1,24 @@
+const controller = require("@controllers/reports/rpt_PA_ChangeRequestTypesFYSummary.js");
+// const validators = require("@validators/report");
+const what = "report";
+
+const routes = [
+  {
+    method: "GET",
+    url: `/${what}/projects/rpt_PA_ChangeRequestTypesFYSummary`,
+    handler: controller.rpt_PA_ChangeRequestTypesFYSummary,
+    // uncomment this when the template is built.
```

Comment:
is the template not build?

Reply:
the template is part of the next ticket. this is the query. sorry for the confusion!